CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_salesforce 0.7.0
+🎉 Salesforce Package Updates 🎉
+
+We are updating the Salesforce package! To improve its utility, the changes include the following:
+## Features
+  - Creating a new Contact Enhanced, Daily Activity, and Opportunity Line Item Enhanced model as well as updating the Opportunity Enhanced model.
```

Comment:
Minor request, can you add the model names to match what the user will see in their warehouse. So "Contact Enhanced" should be `salesforce__contact_enhanced`.